### ISSUE TITLE
Break out tests that require sequencing into separate test suite

### ISSUE THREAD

**Issue by sjezewski** [pachyderm/pachyderm#1051]
We have a set of tests (e.g. all the integration tests that require restart) that _must_ be run sequentially.

I think these are disabled right now, but we should use them if we have them. After #1050 is done, this will be easy to do. This will be the only suite that will be run as a separate make task, to guarantee that they're run in sequence. (Additionally we'll set the right go test flags)

**Comment by jdoliner** [comment]
I think we can just get rid of those tests instead. Given that the server is stateless now testing restarts doesn't really do much for us and it seems like a bit of a pain. If we wanted to have meaningful restart tests it would be a lot more meaningful to restart the database than pachd itself.

**Comment by sjezewski** [comment]
no longer relevant ... all integration tests are run in sequence now
